### PR TITLE
Updating default webcam resolution 

### PR DESCRIPTION
**Pull request by raymondlo84**
Adding resolution config so we will use HD resolution images for webcam demo.

### CODE REVIEW

File: utils/notebook_utils.py [hunk 1]
```diff
@@ -252,6 +252,10 @@ def __init__(self, source, size=None, flip=False, fps=None, skip_first_frames=0)
 
         self.cv2 = cv2  # This is done to access the package in class methods
         self.__cap = cv2.VideoCapture(source)
+        #try HD by default to get better video quality 
+        self.__cap.set(cv2.CAP_PROP_FRAME_WIDTH, 1280)
```
Comment (eaidova) [inline utils/notebook_utils.py]: should it be configurable with default values? What if user want different resolution or use different webcam?

Comment (raymondlo84) [reply utils/notebook_utils.py]: Good point. Updated.

**Review by raymondlo84 — COMMENTED** [review]
Like this? :)

File: utils/notebook_utils.py (hunk 1 above)
Comment (raymondlo84) [reply utils/notebook_utils.py]: https://github.com/openvinotoolkit/openvino_notebooks/pull/2326/commits/d1967a746ec76214b3f6194fd80d2f1219fd4243 =)

**Review by eaidova — APPROVED** [review]

**Pull request merged**